src/Tradesy/Innobackupex/S3/Local/Download.php

```diff
@@ -66,19 +67,29 @@ public function testSave()
 
     public function load(\Tradesy\Innobackupex\Backup\Info $info, $filename)
     {
+        $path_to = $info->getBaseBackupDirectory() . DIRECTORY_SEPARATOR;
+
         //$filename = $info->getLatestFullBackup();
         LogEntry::logEntry('Downloading ' . $filename);
-        LogEntry::logEntry('Saving to: '  . $info->getBaseBackupDirectory() . DIRECTORY_SEPARATOR);
+        LogEntry::logEntry('Saving to: '  . $path_to);
         try {
             $this->client->downloadBucket(
-                $info->getBaseBackupDirectory() . DIRECTORY_SEPARATOR . $filename,
+                $path_to . $filename,
                 $this->bucket,
                 DIRECTORY_SEPARATOR . $info->getRepositoryBaseName() . DIRECTORY_SEPARATOR . $filename,
                 [
                     "allow_resumable" => false,
                     "concurrency" => $this->concurrency,
                     "base_dir" => $info->getRepositoryBaseName() . DIRECTORY_SEPARATOR . $filename,
-                    "debug" => true
+                    "debug" => true,
+                    "before" => function(Command $command) use ($path_to) {
+                        // extract file name from key
+                        $parts = explode('/', $command['Key']);
```

Comment:
Should `/` be `DIRECTORY_SEPARATOR`?

Reply:
`basename` [1] might provide what we need too

[1] https://secure.php.net/manual/en/function.basename.php